Client.xcodeproj/project.pbxproj

```diff
@@ -12404,6 +12404,14 @@
 				minimumVersion = 0.9.11;
 			};
 		};
+		277046C02684CD1E001CB097 /* XCRemoteSwiftPackageReference "PanModal" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/brave/PanModal";
+			requirement = {
+				branch = master;
+				kind = branch;
+			};
+		};
```

Comment:
Let me know if you'd rather me push a new tag and still version the package or use hardcoded commit SHA

Reply:
it's our repo we can just fetch from `master`,
unfortunate that we have to fork it :(

Reply:
At least I can go apply my other changes as well that never got reviewed… 🤷‍♂️

Reply:
Im gonna actually use a SHA ref here, as if I do apply my other fix I'd rather make a new PR for it instead of it magically appearing